rust/agama-lib/share/profile.schema.json

```diff
@@ -188,17 +201,15 @@
                   }
                 }
               }
-            },
-            "required": [
-              "id"
-            ]
+            }
           }
         }
       }
     },
     "user": {
       "title": "First user settings",
       "type": "object",
+      "additionalProperties": false,
```

Comment:
feature request: I am surprised that we don't allow hashed passwords in the profile

Reply:
Yes, you are right, that's something we need to implement. I have created a placeholder card in Trello.